modules/electromagnetics/test/tests/kernels/vector_helmholtz/vector_ADmaterial_wave_equation_convergence.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python3
+# Script to run spatial convergence study on vector_ADmaterial_wave_equation.i
+
+import mms
+import sympy
+import os
+
+if os.path.isfile('../../../../electromagnetics-opt'):
+    executable = '../../../../electromagnetics-opt'
+else:
+     executable = '../../../../../combined/combined-opt'
```

Comment:
@milljm because of this the tests have to be specified as `installation_.. = in_tree` right?

Reply:
For reference, I based this format off of [`/code_verification/mes_spatial.py`](https://github.com/idaholab/moose/blob/next/modules/heat_transfer/test/tests/code_verification/mes_spatial.py) in the Heat Transfer module.

Reply:
seems it's not needed there. So we should be fine